modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/css/_variations-nav.scss

```diff
@@ -23,6 +24,7 @@
 
 	&__label {
 		&.autofit-col {
+			flex: auto;
```

Comment:
I have checked that changing the class `flex-grow-1` to `flex-shrink`-1 in this line has the same effect and looks cleaner https://github.com/liferay-tango/liferay-portal/blob/64e30a3d6793ecde6b0c64840870872978fbd877/modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/js/components/VariationsNav/SortableListItem.js#L110 bit I'm not sure if this can have any undesirable effect.

Reply:
Hey @claraizquierdo 
How did you exactly used the class? I tried but it does not behave as you comment:
<img width="1478" alt="image" src="https://user-images.githubusercontent.com/19485114/178429678-86760585-4da1-4167-8b88-9a54e398546f.png">

Reply:
That grow class was added to solve [this bug](https://issues.liferay.com/browse/LPS-150623) but I think it's no longer needed as the children with flex auto and the link with the flex grow are doing the same job. I'll check this